src/test/java/com/suse/saltstack/netapi/client/SaltStackClientTest.java

```diff
@@ -213,4 +213,24 @@ public void testStartCommand() throws SaltStackException {
         assertEquals(job.getJid(), "20150211105524392307");
         assertEquals(job.getMinions(), Arrays.asList("myminion"));
     }
+    
+    @Test
+    public void testStartCommandAsync() throws SaltStackException {
+    	 stubFor(post(urlEqualTo("/minions")).willReturn(
+                 aResponse().withStatus(HttpURLConnection.HTTP_OK)
+                         .withHeader("Content-Type", "application/json")
+                         .withBody(JSON_START_COMMAND_RESPONSE)));
+
+        Future<Job> future = client.startCommandAsync("*", "test.ping", null, null);
+        Job job;
+        try {
+        	job = future.get();
+        } catch (InterruptedException | ExecutionException e) {
+            throw new SaltStackException(e);
```

Comment:
Why do you wrap `Exception`s here? I would just let the test fail on any `Exception` - wrapping them serves no purpose and adds one stack trace call to look at during debugging. Is there anything I am missing?